tools/tool-testing/selftest.js

```diff
@@ -27,27 +27,14 @@ var release = require('../packaging/release.js');
 var projectContextModule = require('../project-context.js');
 var upgraders = require('../upgraders.js');
 
-require("../tool-env/install-runtime.js");
-
-function checkTestOnlyDependency(name) {
-  try {
-    var absPath = require.resolve(name);
-  } catch (e) {
-    throw new Error([
-      "Please install " + name + " by running the following command:",
-      "",
-      "  /path/to/meteor npm install -g " + name,
-      "",
-      "Where `/path/to/meteor` is the executable you used to run this self-test.",
-      ""
-    ].join("\n"));
-  }
+import { ensureDependencies } from '../cli/dev-bundle-helpers.js';
 
-  return require(absPath);
-}
+const DEV_DEPENDENCY_VERSIONS = {
+  'phantomjs-prebuilt': '2.1.14',
+  'browserstack-webdriver': '2.41.1',
```

Comment:
Pinning these versions is a big win for reproducibility, compared to the current approach of always installing the latest versions.